app.js

```diff
@@ -206,6 +206,22 @@ app.get('/',
         middleware.setUserIfTokenExists,
         routes.index );
 
+app.get('/initializeProject',
+        middleware.checkForAuth,
+        routes.getProject);
+
+app.get('/project/:projectId', routes.openProject);
+
+app.get('/projectExists/:projectName', routes.projectExists);
+
+app.get('/newProject/:projectName', routes.newProject);
+
+app['delete']('/deleteProject/:projectId', routes.deleteProject);
```

Comment:
If you're concerned about the keyword don't be. `app.delete()` should work fine.